proto/blueprint.proto

```diff
@@ -122,6 +123,11 @@ message List {
    */
   optional string web_uri = 14;
   AdTargetingParams ad_targeting_params = 15;
+  /**
+   * Needed on lists and articles. Collections use the adUnit defined
+   * in the fronts response (e.g uk/fronts/home)
+   */
+  string ad_unit = 16;
```

Comment:
We always have this data `ad_unit` for a list?  I just notice that the `ad_unit` in `Article` is optional.

Reply:
Yes so ad_unit on lists will always be the ID of the list, so therefore I made it non-optional as we will always have an ID.

 Conversely, on articles it will be the section ID. However, the section ID itself is an `Option` and so could be `None`. See [this line of code](https://github.com/guardian/mobile-apps-api/blob/b97445e39be5f5099c4c76dfe8568607e3ff5fc4/common-play/src/main/scala/common/models/commercial/CommercialProperties.scala#L20)